usecase/usecase-embedding-android-tests/testscripts/PauseJsTimers.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Embedding api usecase tests
+    Scenario: Check xwalk pause & resume js time
+        When I launch "usecase-embedding-android-test" with "org.xwalk.embedded.api.sample" and "PauseTimersActivity" on android
+        And I register watcher "ClearInfoWindow" when "Info" click "confirm"
```

Comment:
Please format the lines, "And" should be one space interval against "When" and "Then"